template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hive/HiveKnoxConfigProvider.java

```diff
@@ -25,15 +27,20 @@ public List<ApiClusterTemplateConfig> getServiceConfigs(CmTemplateProcessor temp
         String filePerEvent = ConfigUtils.getSafetyValveProperty("hive.hook.proto.file.per.event", "true");
 
         String cdhVersion = templateProcessor.getVersion().orElse("");
+        List<ApiClusterTemplateConfig> configs = new ArrayList<>();
         if (isVersionNewerOrEqualThanLimited(cdhVersion, CLOUDERAMANAGER_VERSION_7_1_0)) {
-            return List.of(config(HIVE_SERVICE_CONFIG_SAFETY_VALVE, filePerEvent));
+            configs.add(config(HIVE_SERVICE_CONFIG_SAFETY_VALVE, filePerEvent));
         } else {
             KerberosConfig kerberosConfigOpt = templatePreparationObject.getKerberosConfig().get();
             String realm = kerberosConfigOpt.getRealm();
             String principal = ConfigUtils.getSafetyValveProperty("hive.server2.authentication.spnego.principal", "HTTP/_HOST@" + realm);
             String keytab = ConfigUtils.getSafetyValveProperty("hive.server2.authentication.spnego.keytab", "hive.keytab");
-            return List.of(config(HIVE_SERVICE_CONFIG_SAFETY_VALVE, principal + keytab + filePerEvent));
+            configs.add(config(HIVE_SERVICE_CONFIG_SAFETY_VALVE, principal + keytab + filePerEvent));
         }
+        if(isVersionNewerOrEqualThanLimited(cdhVersion, CLOUDERA_STACK_VERSION_7_2_10)){
+            configs.add(config(HIVE_SERVICE_CONFIG_SAFETY_VALVE, "<property><name>hive.metastore.try.direct.sql.ddl</name><value>true</value></property><property><name>hive.metastore.try.direct.sql</name><value>true</value></property>"));
+        }
+        return configs;
```

Comment:
Why is this change needed for changing the HMS configuration for Md Ha blueprints with profilers?